### ISSUE TITLE
Latest release 2018.3.0f2 fails to download

### ISSUE THREAD

**Issue by skjalgsm** [sttz/install-unity#20]
The code I'm running provides this output

```
install-unity install 2018.3 -p Unity -p iOS -p Android

Selected 2018.3.0f2 (6e9a27477296) for input 2018.3f
VersionPlatformMetadata.iniUrl is not set for macOS
Parameter name: metadata
```

**Comment by R4yGM** [comment]
try delete unity and reinstall some times

**Comment by sttz** [comment]
Unity changed their URLs again so I'll have to adjust the scraping. But they also have a typo on the archive page (it links to `downoad.unity3d.com` instead of `down*l*oad.unity3d.com`), so automatic scraping won't work until they fix the link.

In the meantime, you can use the release notes url instead of a version number:

    install-unity install https://unity3d.com/unity/whats-new/unity-2018.3.0 -p Unity -p iOS -p Android

**Comment by sttz** [comment]
Just released version 2.1.0, that now uses the `unityhub://` urls when scraping, which will hopefully prove to be more reliable than the old urls Unity is always changing and slowly removing.

The new version now discovers 2018.3.0f2 and 2018.2.20f1.
